viewer/packages/sector-parser/src/GltfSectorParser.ts

```diff
@@ -240,7 +240,7 @@ export class GltfSectorParser {
     if (dracoCompression !== undefined) {
       const dracoBufferView = json.bufferViews[dracoCompression.bufferView];
 
-      const dracoMeshOffset = offsetToBinChunk + dracoBufferView.byteOffset;
+      const dracoMeshOffset = offsetToBinChunk + dracoBufferView.byteOffset!;
```

Comment:
Question for @christjt : Should `byteOffset` be an optional field? If it actually is optional, we should probably fix this in another way.

Reply:
The gltf sharp writer simply will not write the byteOffset field into the gltf file if it has the value of 0. So for our case it is possibly undefined. Agree that it is ugly though and `byteOffset ?? 0` makes a lot more sense.